.travis.yml

```diff
@@ -12,12 +12,14 @@ rvm:
   - jruby-9.1.9.0
   - ruby-head
   - jruby-head
+  - truffleruby
 
 matrix:
   allow_failures:
     - rvm: ruby-head
     - rvm: jruby-head
     - rvm: jruby-9.1.9.0
+    - rvm: truffleruby
```

Comment:
It should not be in `allow_failures`, otherwise nobody will notice it fails typically.
This might have shown that adding RuboCop would break the TruffleRuby build for https://github.com/hanami/utils/pull/344